content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: delfin
+description: "delfin deployment on kubernetes guide."
+tags: ["installation guide", "delfin"] 
+---
+
+## Delfin deployment on kubernetes cluster
+Here are the steps to deploy delfin on a kubernetes  cluster 
+
+### Pre-config (Ubuntu 18.04)
+The installation is tested on Ubuntu 18.04, please make sure to use the right one.
```

Comment:
What is the right one? Is there any specific major/minor version of Ubuntu 18.04 to be used. Please elaborate the minimum requirements.

Reply:
elaborated in the upcoming lines